clang/test/Sema/switch-availability.c

```diff
@@ -15,7 +15,7 @@ enum SwitchTwo {
 };
 
 void testSwitchTwo(enum SwitchTwo st) {
-  switch (st) {} // expected-warning{{enumeration values 'Vim' and 'Emacs' not handled in switch}}
+  switch (st) {} // expected-warning{{enumeration values 'Ed', 'Vim' and 'Emacs' not handled in switch}}
 }
 
```

Comment:
We should have some test showing that the statement after the switch is considered reachable. You can build one with -Wreturn-type and an int return:
```
int doswitch(SwitchTwo e) {
  switch (e) {
  case Emacs: return 42;
  }
} // expected-warning {{asdf}}
```

Reply:
Done.